ludwig/automl/utils.py

```diff
@@ -150,3 +152,23 @@ def _add_option_to_evaluate(
             if option_val not in hyperopt_params[option_param]["categories"]:
                 bisect.insort(hyperopt_params[option_param]["categories"], option_val)
     return point_to_evaluate
+
+
+# Set trainer and hyperopt metric and goal for first output feature if not already set
+def set_output_feature_metric(base_config):
```

Comment:
nit: Could we fold in the `== 1` logic into this function:

```
if len(base_config["output_features"]) > 1:
    # Multiple output features. Rely on default minimize `combined.loss goal`.
    # In the future, we could be more clever about choosing a combination of metrics.
    return
# if single output feature, set relevant metric and goal if not already set
    ...
```

Reply:
done.